mlflow/pyfunc/__init__.py

```diff
@@ -1067,7 +1067,8 @@ def _predict_row_batch(predict_fn, args):
                 names = [str(i) for i in range(len(args))]
             else:
                 names = input_schema.input_names()
-                required_names = input_schema.required_input_names()
+                # required_names = input_schema.required_input_names()
+                required_names = names
```

Comment:
I used MLR 13.1. MLflow 2.3.1 is installed there, but mlflow in my machine is 2.4.2.dev0. 2.3.1 doesn't have the `required_input_names` attribute.